AOJ/06/06_C.py

```diff
@@ -0,0 +1,20 @@
+n = int(input())
+check = [[[0 for i in range(10)]  for j in range(3)] for k in range(4) ] 
+x = 0 
+
+while n != 0:
+    b,f,r,v = map(int,input().split())
+    check[b - 1][f - 1][r - 1] += v
+    n -= 1
```

Comment:
`while`のループ回数が`n`と決まっているため、`for`ループでかけます。
```suggestion
for _ in range(n):
    b,f,r,v = map(int,input().split())
    check[b - 1][f - 1][r - 1] += v
```